lib/quby/answers/services/updates_answers.rb

```diff
@@ -49,6 +46,20 @@ def on_failure(&block)
 
         protected
 
+        def save_raw_params_on_error(new_attributes, &_block)
```

Comment:
Je hebt wel een punt dat het duidelijker is om een duidelijk  `rescue`-pad te hebben voor "stel could_not_update_at in bij exceptie" en een happy path in de yield.